docs/pricing.md

```
@@ -1,32 +1,102 @@
-- **Private Server**: Using a self-hosted private server gives you the ability to send an unlimited number of messages. However, since push notifications are sent through our cloud server, there is a rate limit on our side. You may need to adjust the notification debounce period in your private server's configuration to accommodate this.
+- :material-cellphone-check: [**Local Server**](./getting-started/local-server.md)  
+    - 🆓 Completely free  
+    - 🚫 No restrictions or limits  
+    - 🔄 Direct device communication  
+    - :material-connection: No internet required
 
-## Future Guarantees
+- :material-cloud: [**Cloud Server**](./getting-started/public-cloud-server.md)  
+    - 🆓 Completely free  
+    - ⚠️ Individual limits applied only if affecting other users  
+    - 📈 Recommended to notify [support](support@sms-gate.app) for >10k/day usage  
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)